OWNERS

```diff
@@ -1,44 +1,12 @@
 approvers:
-- cagiti
-- ccojocar
-- daveconde
-- deanesmith
+- ankitm123
+- babadofar
 - dependabot
```

Comment:
remove dependabot, it wont do anything, as it's not in jx org, so it cant even trigger a job, and anyways it's a bot, so cant approve.